sys/include/net/sock/dtls.h

```diff
@@ -491,8 +491,25 @@
 extern "C" {
 #endif
 
+/**
+ * @ingroup net_sock_util_conf
+ * @{
+ */
+/**
+ * @brief   Default buffer size for DTLS handshake (as exponent of 2^n)
+ *
+ * As the buffer size ALWAYS needs to be power of two, this option represents
+ * the exponent of 2^n, which will be used as the size of the buffer
+ * ( @ref DTLS_HANDSHAKE_BUFSIZE ).
+ */
+#ifndef CONFIG_DTLS_HANDSHAKE_BUFSIZE_EXP
+#define CONFIG_DTLS_HANDSHAKE_BUFSIZE_EXP  8
+#endif
+/** @} */
+
```

Comment:
Maybe it is better here to use the other documentation style (`/** ... */`)? It looks a bit odd now that the macro is longer and the first line does not fit.

Reply:
Squashed. Modified documentation for `SOCK_DTLS_HANDSHAKE` as well.